WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -1,39 +1,48 @@
 import SwiftUI
 
 /// iOS type style definitions for POS
-/// TfaZ4LUkEwEGrxfnEFzvJj-fi-3385_18076
+/// 1qcjzXitBHU7xPnpCOWnNM-fi-23_7310
 enum POSFontStyle {
-    case posTitleRegular
-    case posTitleEmphasized
-    case posBodyRegular
-    case posBodyEmphasized
-    case posLargeDetailEmphasized
-    case posLargeDetailRegular
-    case posDetailLight
-    case posDetailRegular
-    case posDetailEmphasized
+    case posHeading
+    case posBodyXLarge
+    case posBodyLargeBold
+    case posBodyLargeRegular
+    case posBodyLargeRegularUnderline
+    case posBodyMediumBold
+    case posBodyMediumRegular
+    case posBodyMediumRegularUnderline
+    case posBodySmallBold
+    case posBodySmallRegular
+    case posBodySmallRegularUnderline
+    case posCaptionBold
+    case posCaptionRegular
     case posButtonSymbol
 
     func font(maximumContentSizeCategory: UIContentSizeCategory? = nil) -> Font {
         switch self {
-        case .posTitleRegular:
-            Font.system(size: scaledValue(36, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .medium)
-        case .posTitleEmphasized:
-            Font.system(size: scaledValue(36, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .bold)
-        case .posBodyRegular:
-            Font.system(size: scaledValue(24, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posBodyEmphasized:
-            Font.system(size: scaledValue(24, maximumContentSizeCategory: maximumContentSizeCategory), weight: .bold)
-        case .posLargeDetailEmphasized:
-            Font.system(size: scaledValue(20, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
-        case .posLargeDetailRegular:
-            Font.system(size: scaledValue(20, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posDetailLight:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .regular)
-        case .posDetailRegular:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .medium)
-        case .posDetailEmphasized:
-            Font.system(size: scaledValue(16, maximumContentSizeCategory: maximumContentSizeCategory), weight: .semibold)
+        case .posHeading:
+            Font.system(size: scaledValue(FontSize.heading, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge), weight: .bold)
+        case .posBodyXLarge:
+            Font.system(
+                size: scaledValue(FontSize.bodyXLarge, maximumContentSizeCategory: maximumContentSizeCategory ?? .accessibilityLarge),
+                weight: .semibold
```

Comment:
Should it be regular?

```suggestion
                weight: .regular
```

Reply:
I used `semibold` because the weight is 590 vs. 400 in regular and 700 in bold:

<img width="821" alt="Screenshot 2025-02-12 at 8 49 00 AM" src="https://github.com/user-attachments/assets/2e2a6d8d-e344-4da1-a890-72ce4554779c" />

Let me know if you think `regular` is the way to go and I can make the change separately, maybe the mapping is different by font type.

Reply:
No, all good, I checked the wrong font on Figma!